Fixes 6431 - Updates about_Splatting (#6473)
MicrosoftDocs/PowerShell-Docs: reference/7.1/Microsoft.PowerShell.Core/About/about_Splatting.md

position or by parameter name. Also, you can splat multiple objects in a single
command so you don't pass more than one value for each parameter.

> As of PowerShell 7.1, you can also override a splatted parameter by explicitly

Reviewer: Is also needed?  Feels like a filler word.

Author: good point. Fixed

## Examples

### Example 1

> 

Reviewer 2: Need to add descriptive titles to the example H3 lines

Author: Fixed

> ### Example 1 Reuse splatted parameters in different commands

Reviewer: Missing `:` after `Example #`

Author: Fixed

Write-Host @Colors "This is another test."
```

> ### Example 2 Forward parameters using $PSBoundParameters

Reviewer: Missing `:` after `Example #`

Author: fixed

3
```

### Example 3: Override splatted parameters with explicitly defined parameters

This example shows how to override a splatted parameter using explicitly
defined parameters. This is useful when you don't want to build a new hashtable
or change a value in the hashtable you are using to splat.

> The `$commonParams` variable stores the parameters to create VMs in the

Reviewer: CSG recommends spelling out VMs.

Author: fixed

`East US` location. The `$allVms` variable is a list of VMs to create. We loop
through the list and use `$commonParams` to splat the parameters to create each
VM. However, we want `myVM2` to be created in a different region than the other

> VMs. Instead of adjusting the `$commonParams` hashtable, we can now explicitly

Reviewer 2: ```suggestion
VMs. Instead of adjusting the `$commonParams` hashtable, you can explicitly
```

Author: Fixed

define the **Location** parameter in `New-AzVm` to supersede the value of the
`Location` key in `$commonParams`.

```powershell
$commonParams = @{
    ResourceGroupName = "myResourceGroup"
    Location = "East US"
    VirtualNetworkName = "myVnet"
    SubnetName = "mySubnet"
    SecurityGroupName = "myNetworkSecurityGroup"
    PublicIpAddressName = "myPublicIpAddress"
}

$allVms = @('myVM1','myVM2','myVM3',)

foreach ($vm in $allVms)
{
    if ($vm -eq 'myVM2')
    {

>         New-AzVm @commonParams -Name $vm -Location "West Us"

Reviewer: Should be `West US`.

Author: Fixed
